cluster-provision/images/vm-image-builder/fedora-realtime/image-url-arm64

```diff
@@ -0,0 +1 @@
+https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/32/Cloud/aarch64/images/Fedora-Cloud-Base-32-1.6.aarch64.qcow2
```

Comment:
Can you update this to F35 at least? The same image as `fedora-kubeadm` & `fedora-with-test-tooling`

```
https://download.fedoraproject.org/pub/fedora/linux/releases/35/Cloud/aarch64/images/Fedora-Cloud-Base-35-1.2.aarch64.qcow2

Reply:
I will update the Fedora OS in a follow-up PR.